libs/gi/ui/src/components/DocumentDisplay.tsx

```diff
@@ -128,6 +137,24 @@ function TextSectionDisplay({ section }: { section: IDocumentText }) {
   const { data } = useContext(DataContext)
   return <div>{evalIfFunc(section.text, data)}</div>
 }
+function TextSectionDisplayCollapse({ section }: { section: IDocumentText }) {
+  const { data } = useContext(DataContext)
+  const [expanded, setExpanded] = useState(false)
+  return (
+    <Collapse
```

Comment:
Also I think we could add a small hint dropdown arrow so it is more obvious it can be expanded 
<img width="205" alt="image" src="https://github.com/frzyc/genshin-optimizer/assets/36019388/22ebc300-0d4c-4c25-8679-56e2aa8ae73e">
Especially for smaller text sections, it just looks like the site has some weird styling, rather than an expandable description